modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -406,7 +408,7 @@ this option, the 3rd party app needs to show the password as plaintext
 on standard output.
 
 [[rd-party-software-examples]]
```

Comment:
if an explicit anchor's desirable, I'd suggest it should be `3rd-party-software-examples` (unless that doesn't work).

Reply:
I did not set the anchor, it was already there. We need to fix such things in own pr´s